src/main/java/no/nav/veilarbaktivitet/controller/AktivitetsplanController.java

```diff
@@ -83,6 +83,25 @@ public AktivitetDTO oppdaterAktivitet(@RequestBody AktivitetDTO aktivitet) {
                 .orElseThrow(RuntimeException::new);
     }
 
+    @PutMapping("/{id}/kanDeleCV")
+    public AktivitetDTO oppdaterKanCvDeles(@RequestBody AktivitetDTO aktivitet) {
```

Comment:
er det ikke greiere og lage en kustom dto. burde ikke sende inn hele aktiviteten.
Burde vel også legges i mappen for delingavCv

Reply:
eks på dto:
svar, version.

Reply:
Burde dette ligge i en egen fil og mappe? Litt sånn som vi gjorde med forhåndsorientering?

Reply:
Ja, jeg lagde ikke egen mappe fordi det er laget en mappe i den andre pr'en som dere jobber med. Virka som det tok litt tid med den så jeg bare lagde en pr :)

Reply:
Lurte på å ha en minimal DTO, men ble usikker fordi det skal insertes ny instans i Aktivitettabellen så beholdt det i tråd med det som var fra før. oppdaterEtikett og oppdaterStatus tar også hele AktivitetDTO